public/js/search.js

```diff
@@ -1,15 +1,96 @@
 const handleSearch = async () => {
+  console.log("handleSearch called");
```

Comment:
Missing prevent default, but button isn't in form so shouldn't be a problem for now.